openzeppelin/token/erc721/ERC721_Mintable_Pausable.cairo

```diff
@@ -3,7 +3,7 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.cairo.common.uint256 import Uint256
 
-from contracts.token.ERC721_base import (
+from openzeppelin.token.erc721.library.ERC721_base import (
```

Comment:
```suggestion
from openzeppelin.token.erc721.library import (
```

the idea is to go with `library` directly, no more references to "base contracts"

Reply:
Ahhhh I see. On it!